cayleypy/graphs_lib.py

```diff
@@ -600,6 +600,73 @@ def consecutive_k_cycles(n: int, k: int) -> CayleyGraphDef:
             name=name,
         )
 
+    def sheveleva2(n: int, k: int) -> CayleyGraphDef:
+        """
+        Sheveleva generators - consists only of 2 elements, and they have two parameters n, k,
+        where n - length of permutations, k - additional parameter – position of the square.
+        """
+        assert k >= 1 and k <= n-3, "k must be >= 1 and <= n-3"    
+        p1 = [i for i in range(n)]
+        for i in range(0,(n//2)*2):
+            if i % 2 == 0:
+                p1[i] = i + 1
+            else:
+                p1[i] = i - 1
+        p2 = [i for i in range(n)]
+        for i in range(0,((n-1)//2)*2):
+            if (i+1) % 2 == 1:
+                p2[i+1] = i + 2
+            else:
+                p2[i+1] = i 
```

Comment:
Again, it looks like these are transpositions (1,2), (3,4)...

```
p2 = permutation_from_cycles([[i, i+1] for i in range(1,n,2)])
```

But please check if I am correct.